homework/romanus_yaroslav/week6/report.md

```diff
@@ -0,0 +1,96 @@
+### Week 6 Regular Expressions Homework
+
+### This is a template for the report of 21 problems on [this site](http://regextutorials.com/excercise.html).
+
+### Fork this repository, change this file and send us the link for grading.
+
+### We recommend you (re)watch our presentation, and use a [cheatsheet](./cheatsheet.md) while working on the following problems.
+
+### If you are having problems with figuring out the correct solution, skip it for later, and if you are completely lost, use the hints and try to understand them.
+
+---
+
+### Problem 1: Floating point numbers
+```
+\d+(\.\d+)
+```
+### Problem 2: Years before 1990
+```
+.*19[1-8]\d.*
+```
+### Problem 3: Hexadecimal colors
+```
+#.{6}
```

Comment:
I guess this works, but this will match literally anything starting with a `#`. Please try to make it more concrete!